web-jehlomat/src/config/user.ts

```diff
@@ -0,0 +1,31 @@
+import jwt_decode from "jwt-decode";
+import { AxiosResponse } from 'axios';
+import {API} from './baseURL';
+
+interface IToken {
+    "aud": string;
+    "user-id": string;
+    "iss": string;
+    "exp": string;
+}
+
+export interface IResponse {
+    id: number;
+    email?: string;
+    username: string;
+    password?: string;
+    organizationId: number;
+    teamId: number;
+    isAdmin?: boolean;
+    verified?: boolean;
+}
+
+export const getMe = (token: string) => {
+    const decoded: IToken = jwt_decode(token);
+    return decoded["user-id"];
+}
+export const getUser = async (token: string) => {
```

Comment:
this whole file shouldn't be needed at the moment, you can use user that is already fetched from state like this `const user = useRecoilValue(userState);`